OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -117,7 +119,6 @@ public bool Disabled
 					return;
 
 				disabled = value;
-				Invalidate(map.ProjectedCellBounds);
```

Comment:
This is only used by the debug command and that explicitly calls `ExploreAll` and `ResetExploration`, so this is not needed.

Reply:
Can we leave a comment here to that effect? If anything else ever ends up using this - we'll need to change it.

Reply:
It seems I got mixed up here. The explore methods are not called by the cheat command to disable visibility, they are only relevant for resetting the shroud.

The relevant cheat command disables fog rendering simply by setting the `RenderPlayer` to null, which is the same thing as what happens when you choose to disable fog as a spectator. The only thing that happens by setting `Shroud.Disabled` to true is that it enables a blanket shortcut in the `IsExplored` and `IsVisible` methods. This ensures that visibility checks are circumvented for all units under the players command. There is no need to invalidate any cells, because that information is just completely bypassed in the checks.

Reply:
:+1: